README.md

```diff
@@ -6,22 +6,140 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
+
+-   **Healthcheck** - service and cronjob health checks to see if yangcatalog is working correctly
+-   **Logs** - reading and filtering through different log files of yangcatalog.org
+-   **Users** - reading and updating yangcatalog user database and granting them rights
+-   **Config** - reading and updating yangcatalog config file
+-   **Nginx** - reading yangcatalog nginx files
+-   **Files** - reading updating and deleting files from main yangcatalog folder
+-   **Actions** - running scripts (setting their options) and checking for their status
+
+Following deeper description of each tab and instructions on how to use them
+
+## Healthcheck
+
+Healthcheck tab can be divided into two parts:
+
+-   **Services Healthcheck**
+-   **Cronjobs Healthcheck**
+
+### Services healthcheck
+
+Services healthcheck check status of each of the running services at one minute intervals. By status check we mean sending a real request, which is processed by the service and the result is send back (e.g. validate RFC by YANG validator, or search for a specific module by YANG search etc.). The status of each service can acquire following 3 states:
+
+-   **down** - Service is down and it is not possible to process the request.
+-   **problem** - Service is running, but some problem occured during processing the request.
+-   **running** - Service is running and the request was processed successfully.
+
+### Cronjob healthcheck
+
+Unlike services healthcheck, the cronjobs healthcheck are not refreshed at one minute intervals. They simply show the status of the last cronjob run. Most of them are performed once a day, so there is no need for refresh. The status of each cronjob can acquire following 2 states:
+
+-   **Fail** - Error occurred while running the script and the script did not finished successfully.
+-   **Success** - The script ran successfully, without errors.
+
+### Usage
+
+This tab has mostly informative value, the background colors of individual headings corresponds to the status of service/cronjob runs (the same colors are used as for the traffic light). However, it is possible to click on reload button at services, to send request to the service manually. Also, if an error occurs while running the cron job, a small icon with letter "i" is displayed next to the name of job. After clicking on the icon, an error message will be displayed in the modal window.
+
+## Logs
+
+The logs tab is used to display and filter log messages from selected log files. Log messages are displayed from the newest to the oldest.
+It is possible to display log messages from several log files at once. In this case the logs from first log file are displayed, followed by logs from second file etc.
+
+### Usage
+
+The filter provides several properties according to which log messages can be filtered. Each of this properties has default value in the case that user does not set its value. After clicking **Filter** button, log messages are displayed in table under the filter.
```

Comment:
provide information on what kind of filtering we provide. just briefly describe each of the